currency-converter/currency.js

```diff
@@ -1,80 +1,163 @@
 document.addEventListener("DOMContentLoaded", () => {
-  const rates = {
-    timestamp: Date.now(),
-    base: "EUR",
-    date: new Date().toISOString().split("T")[0],
-    rates: {
-      USD: 1.23,
-      GBP: 0.88,
-      EUR: 1.82,
+  const currencyRates = [
```

Comment:
Well done! This part was not super clear from the brief but you understood it well 👍🏻